groups/groups.yaml

```diff
@@ -1349,9 +1349,10 @@ groups:
     settings:
       ReconcileMembers: "true"
     members:
-      - thockin@google.com
-      - bowei@google.com
       - antonio.ojea.garcia@gmail.com
+      - bowei@google.com
+      - cmluciano@us.ibm.com
+      - thockin@google.com
```

Comment:
Not sure how this works - it seems like it should closely match the OWNERS file you're changing below. I think it probably makes sense to expand this list to include any k8s members that are also service-apis approvers - similar to what you've done below in the OWNERS file.

Reply:
Sounds good, I will update this once I get a response from Justin and Harry